### ISSUE TITLE
Nukkit

### ISSUE THREAD

**Issue by DeveloperNeb101** [BentoBoxWorld/BSkyBlock#308]
Does this plugin work on NukkitX?

**Comment by tastybento** [comment]
No, it does not. It would require rewriting to their APIs. I do not know enough about those APIs to know if it would be possible or not.

By the way, I never knew about this project before. I assume it's to enable BedRock client usage?
